main.go

```diff
@@ -0,0 +1,32 @@
+// Copyright Contributors to the Open Cluster Management project
+
+package main
+
+import (
+	"flag"
+
+	"github.com/open-cluster-management/search-indexer/pkg/config"
+	"github.com/open-cluster-management/search-indexer/pkg/server"
+	"k8s.io/klog/v2"
```

Comment:
Updated the logger from glog.
```
The decision to create klog was one that wasn't made lightly, but it was necessary due to some drawbacks that are present in glog. Ultimately, the fork was created due to glog not being under active development;
```
https://pkg.go.dev/k8s.io/klog/v2#section-readme